docs/source/faqs.md

```diff
@@ -0,0 +1,15 @@
+(faqs)=
+
+# Frequently Asked Questions
+
+```{list-table}
+:header-rows: 1
+:name: tab-faqs
+
+* - Question
+  - Answer
+* - Is SATRE an ISO technical standard?
+  - No. The SATRE specification aims to provide a helpful guide for TRE Operators, Developers and Builders. It can be used to evaluate existing TREs and inform how they could be improved, or inform the development process of new TREs.
```

Comment:
```suggestion
  - No. The SATRE specification aims to provide a helpful guide for TRE Operators, Developers and Builders. It can be used to inform the development process of new TREs, or to evaluate existing TREs and inform how they could be improved. Meeting the SATRE specification standards may help to demonstrate how your TRE complies with standards such as ISO 27001.
```

Reference to a specific ISO standard might be pushing it a bit, but I do think a helpful aim of the spec should be to help people meet such standards, even though SATRE itself carries no legal weight.

Reply:
I think it's the other way round- complying with ISO27001 makes it easier to comply with SATRE.